### PR TITLE
BPTL Dashboard: Update home collection derived variables upon kit receipt 

### PR DESCRIPTION
**Pull request by abhinavjonnada82**
Derived variables for home collection are updated upon kit receipt in Participants Table

This PR addresses following issues:
https://github.com/episphere/connect/issues/813

### CODE REVIEW

**Review by jhflorey — APPROVED** [review]

**Comment by jhflorey** [comment]
Looks good to me.

**Review by ramavavi — APPROVED** [review]

**Comment by ramavavi** [comment]
Looks good.

File: utils/firestore.js [hunk 1]
```diff
@@ -2281,6 +2281,10 @@ const storeKitReceipt = async (package) => {
         });
 
         await participantDoc.ref.update({
+            '684635302': '375535639',
+            '254109640': '375535639',
```
Comment (JoeArmani) [inline utils/firestore.js:2285]: Just double checking here: the DD has yes/no values listed for these two vars. I'm not sure if we have to update the DD or if this code needs to be:
```javascript
'684635302': '353358909',
'254109640': '353358909',
```

**Review by JoeArmani — COMMENTED** [review]
Just have a question on two values, thanks.

**Review by JoeArmani — APPROVED** [review]
looks good

**Pull request merged**